src/hooks/season/constants.ts

```diff
@@ -1,38 +1,40 @@
 import { DateTime } from 'luxon';
-
-const SEASON_START_DATE = process.env.REACT_APP_COMMUNITY_SEASON_START_DATE
-  ? DateTime.fromISO(process.env.REACT_APP_COMMUNITY_SEASON_START_DATE)
-  : DateTime.now();
-
-export type Season = {
-  id: number;
-  startDate: DateTime;
-  endDate: DateTime;
-};
+import { Season } from './types';
 
 export const SEASONS: Season[] = [
```

Comment:
I am not sure if they should be ENV variables. For me they look like hardcoded objects we keep. If your concern is that people will understand when seasons start and end by looking into source code then yes it makes sense to be injected as ENV var, otherwise it is fine as it is imho.

Reply:
tbh, not really concerned about people looking into source code. let's keep it as is